test/gurka/test_use_tracks.cc

```diff
@@ -22,18 +22,19 @@ class UseTracksTest : public ::testing::Test {
     A-1----B--------E-----2-F
            |        |       |
            |        |       |
-           |        |       3
            |        |       |
-           C--------D       G
```

Comment:
can you briefly describe the test change, its not really obvious from the diff

Reply:
yeah. as far as I introduced `track_penalty`, route `ABEF` became cheaper and route `ABCDEF` became more expensive (because of high penalty in transition cost). So, in order to demonstrate track avoidance feature, we should have route completely without tracks as `ABCDEF` and with tracks as `ABEF`